src/common.props

```diff
@@ -14,6 +14,7 @@
     <HyperionVersion>0.9.8</HyperionVersion>
     <NewtonsoftJsonVersion>12.0.2</NewtonsoftJsonVersion>
     <NBenchVersion>1.2.2</NBenchVersion>
+    <ProtobufVersion>3.9.1</ProtobufVersion>
```

Comment:
Added new entry in `common.props` to standardize Google.Protobuf versions across Akka.Persistence, Akka.Remote, and Akka.Streams.

Akka.Cluster and other packages that depend on Google.Protobuf draw their versions from one of these three dependencies.